### PR TITLE
Added Start Screen

### PR DESCRIPTION
**Pull request by amitsinghsutara**

### CODE REVIEW

File: src/scenes/start-scene.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        const y = event.clientY - rect.top;
+        // console.log(y, this.height / 1.8);
+        if (
+          Math.sqrt(y - this.height / 5.4) < 10 &&
+          Math.sqrt(x - this.width / 2.6) < 10
+        ) {
+          console.log(this.id);
+          self.context.clearRect(0, 0, canvas.width, canvas.height);
+          new LevelSelectionScreen(canvas, data);
+          document.getElementById("playButtonCanvas").style.zIndex = 0;
```
Comment (ashishkrr) [inline src/scenes/start-scene.js:154]: use constants from common.js

File: src/scenes/start-scene.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          document.getElementById(this.id).style.zIndex = 0;
+          document.getElementById("playButtonCanvas").style.display = "none";
```
Comment (ashishkrr) [inline src/scenes/start-scene.js:156]: use constants from common.js

File: src/scenes/start-scene.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+          document.getElementById("monsterCanvas").style.display = "none";
```
Comment (ashishkrr) [inline src/scenes/start-scene.js:157]: use constants from common.js

File: src/components/buttons/play_butoon.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+export default class PlayButton {
```
Comment (ashishkrr) [inline src/components/buttons/play_butoon.js:1]: Where this class being used?

**Review by Rajesh1041 — APPROVED** [review]

**Pull request merged**